mantidproject/mantid
# Move MuonAnalysis documentation to .rst file

Sulaimaan (work experience student at ISIS) created an .rst file for the `MuonAnalysis` interface.

**To test:**

Build and read the documentation for the `MuonAnalysis` interface. It should look similar to the [wiki page](http://www.mantidproject.org/MuonAnalysisSettings).

I've noticed that in the `Home` tab, section `Detector Grouping`, the option to specify a value for `alpha` does not seem to exist any more, so I've removed the section explaining how to use it. This is the only difference with the wiki page.

Fixes #13644.
Fixes #11731

[Release notes](https://github.com/mantidproject/mantid/blob/f51624cf2c51695d82128c9d88efaa5de77893b1/docs/source/release/v3.8.0/muon.rst) have been updated.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
